py_hamt/encryption_codec.py

```
@@ -0,0 +1,64 @@
+from typing import Callable
```

Ruff (0.8.2)

`typing.Callable` imported but unused

Remove unused import: `typing.Callable`

(F401)